pennylane/hf/molecule.py

```diff
@@ -110,3 +110,89 @@ def __init__(
         self.nuclear_charges = [atomic_numbers[s] for s in self.symbols]
 
         self.n_electrons = sum(np.array(self.nuclear_charges)) - self.charge
+
+        self.mo_coefficients = None
+
+    def generate_atomic_orbital(self, index):
+        r"""Return a function that computes the value of a basis function at a given position.
+
+        Args:
+            index: index of the basis function
+
+        Returns:
+            function: function that computes the basis function
+
+        **Example**
+
+        >>> symbols  = ['H', 'H']
+        >>> geometry = np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 1.0]], requires_grad = False)
+        >>> mol = qml.hf.Molecule(symbols, geometry)
```

Comment:
Don't we need `hf.generate_scf(mol)()` here?

Reply:
No, it is needed to obtain the molecular orbital coefficients only.